triage_resolving_filters.json

```diff
@@ -1,5 +1,12 @@
 {
-  "master_failed_to_pull_ignition_signature" : {
-    "MGMT-11089": "Master nodes failed to pull ignition"
+  "signatures": {
+    "master_failed_to_pull_ignition_signature" : {
+      "MGMT-11089": "Master nodes failed to pull ignition"
```

Comment:
AFAIU this message will be looked for in the comments for closing. Is this sitll necessary? Should we get instead the message from the signature class, as [we already have that](https://github.com/openshift-assisted/assisted-installer-deployment/blob/master/tools/triage/add_triage_signature.py#L330)?